tf2_sensor_msgs/package.xml

```diff
@@ -9,23 +9,20 @@
   <license>BSD</license>
 
   <url type="website">http://www.ros.org/wiki/tf2_ros</url>
-    
-  <buildtool_depend version_gte="0.5.6">catkin</buildtool_depend>
 
-  <build_depend>cmake_modules</build_depend>
-  <build_depend>eigen</build_depend>
-  <build_depend>sensor_msgs</build_depend>
-  <build_depend>tf2</build_depend>
-  <build_depend>tf2_ros</build_depend>
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
 
-  <run_depend>cmake_modules</run_depend>
-  <run_depend>eigen</run_depend>
-  <run_depend>sensor_msgs</run_depend>
-  <run_depend>tf2_ros</run_depend>
-  <run_depend>tf2</run_depend>
-  <run_depend>python_orocos_kdl</run_depend>
+  <depend>cmake_modules</depend>
+  <depend>Eigen3</depend>
```

Comment:
I'm not sure capital `Eigen3` is defined in rosdistro.
```
ERROR: the following packages/stacks could not have their rosdep keys resolved
to system dependencies:
tf2_sensor_msgs: Cannot locate rosdep definition for [Eigen3]
```

@clalancette , was this migration intentional?

Reply:
@ruffsl I think you're right, I looked at rosdistro and it looks like `eigen` leads to `eigen3` debians

Reply:
See https://github.com/ros2/geometry2/pull/76

Reply:
I actually don't remember at all, sorry.  However, this package *is* released into Bouncy, so I'm pretty sure what is currently in there works one way or another.